sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -767,6 +723,84 @@ private void processMinidumpFiles() {
                 }
             }
         }
+
+        /* Remove the minidump subfolders from previous sessions. */
+        ErrorLogHelper.removeStaleMinidumpSubfolders();
+    }
+
+    /**
+     * Process the minidump, save an error log with a reference to the minidump, move the minidump file to the 'pending' folder.
+     *
+     * @param minidumpFile    a file with .dmp extension.
```

Comment:
but we filter inside the method.
I believe listFiles already has a filter options and could be used to exclude device info file there.
Also we use exclude rule not include rule so the comment is not really accurate.

Reply:
I've updated code to use a filter inside the listFiles() method. Not sure if I got right the idea about include/exclude rules, let me know if I have to change smth else.